net/ftpproxy/+POST_DEINSTALL

```diff
@@ -0,0 +1,28 @@
+DIFF='--- filter.inc.ftpproxy 2016-09-21 16:38:53.947075272 +0200
```

Comment:
@fbrendel @fichtner I don't think this will survive an upgrade of the opnsense package. If we only need the anchors, we might consider adding them to the core package for the short term and move them out when the plugin system is in place  (https://github.com/opnsense/core/issues/993 ).